README.md

```diff
@@ -49,8 +49,8 @@ Additional CLI options that must be appended to the minio server start command.
 
 Minio access and secret keys.
 
-    skip_server: False
-    skip_client: False
+    minio_skip_server: False
```

Comment:
Can we change this to positive wording to avoid the confusing double negative?

```
minio_install_server: true
minio_install_client: true
```

Reply:
I agree that this would be better